science/2022-11-23-science-blog-6/index_de.md

```diff
@@ -0,0 +1,270 @@
+
+1. [Das Wichtigste in Kürze: Zahlen, Daten, Fakten](#1-das-wichtigste-in-kürze-zahlen-daten-fakten)
+2. [Positivenanteile (bundesweit)](#2-positivenanteile-bundesweit)
+    1. [RAT](#21-rat)
+    2. [PCR](#22-pcr)
+3. [Lokale Unterschiede der Inzidenz](#3-lokale-unterschiede-der-inzidenz)
+4. [Positivenanteile (lokal)](#4-positivenanteile-lokal)
+    1. [RAT](#41-rat)
+    2. [PCR](#42-pcr)
+5. [Positivenanteil und Inzidenz](#5-positivenanteil-und-inzidenz)
+    1. [RAT](#41-rat)
+    2. [PCR](#42-pcr)
+6. [Unterschiede zwischen urbanen und ländlichen Kreisen](#6-unterschiede-zwischen-urbanen-und-ländlichen-kreisen)
+7. [Fazit](#7-fazit)
+8. [So geht es weiter](#8-so-geht-es-weiter)
+
+## 1 Das Wichtigste in Kürze: Zahlen, Daten, Fakten
+
+- Ob Antigen-Schnelltest oder PCR-Test: der Positivenanteil von roten Warnungen ist stets größer als derjenige von grünen Warnungen (niedriges Risiko mit Risikobegegnung). Mit anderen Worten: dort wo Risikobegegnungen erkannt werden, kann die CWA das Risiko zuverlässig einschätzen.
+
+- Für Antigen-Schnelltests ist der Positivenanteil (der Anteil positiver an allen durchgeführten Tests) nach einer roten Warnung zu allen Zeiten größer als die Positivenanteile nach einer grünen Warnung (niedriges Risiko mit Risikobegegnung) bzw. bei niedrigem Risiko ohne Risikobegegnung.
+
+- Für PCR-Tests ist der Positivenanteil nach einer roten Warnung stets größer als der Positivenanteil nach einer grünen Warnung (niedriges Risiko mit Risikobegegnung). In Zeiten einer hohen Inzidenz ist der Positivenanteil nach einer roten Warnung zum Teil kleiner als bei niedrigem Risiko ohne Risikobegegnung – hier ist eine starke Vorselektion durch positive Antigen-Schnelltests zu vermuten.
+
+- Der erhöhte Positivenanteil nach einer roten Warnung im Vergleich zu den Positivenanteilen bei einem niedrigen Risiko mit bzw. ohne Risikobegegnung lässt sich nicht allein mit lokalen Unterschieden der Inzidenz erklären.
+
+- Für Antigen-Schnelltests und PCR-Tests ist die Abhängigkeit eines positiven Testergebnisses von der Inzidenz nach einer roten Warnung nur etwa halb so hoch, wie bei einem niedrigen Risiko ohne Risikobegegnung.
+
+- Die Unterschiede zwischen den Positivenanteilen und damit die Aussagekraft einer roten Warnung hinsichtlich des Infektionsrisikos sind in ländlichen Gebieten deutlicher als in urbanen.
+
+## 2 Positivenanteile (bundesweit)
+
+Täglich spenden etwa 15,4 Mio. Nutzende der CWA ihre Daten. So wurden bisher ca. 37,4 Mio. Testergebnisse übermittelt (Stand: 14. November 2022). Hierbei wurden nur solche Testergebnisse berücksichtigt, die positiv oder negativ ausfielen (keine ausstehenden und ungültigen). Durch die CWA identifizierte Risikobegegnungen wurden nur berücksichtigt, wenn sie bis zu 10 Tage vor der Testregistrierung lag (dies war bei ca. 10,7 Mio. Testergebnissen der Fall). Eine Angabe zum Kreis (Landkreis bzw. kreisfreie Stadt des Wohnorts des CWA-Nutzenden) lag bei ca. 14,3 Mio. Testergebnissen vor.
```

Comment:
"Täglich spenden etwa 15,4 Mio. Nutzende der CWA ihre Daten." That's a nice note, but I'm missing some sort of information like: "Die von diesen Nutzenden gespendeten Daten sind Grundlage für diesen Science-Blog" or something like this.